Source/App/UI/Common/RatingViewController.swift

```diff
@@ -85,70 +140,37 @@ class RatingViewController: UIViewController, UITableViewDelegate, UITableViewDa
 
         return cell
     }
-    
+}
+
+extension RatingViewController: UITableViewDelegate {
+
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+
         let cell = tableView.cellForRow(at: indexPath) as! RatingTableViewCell
         let rowString = String(indexPath.row)
-
         ratingsDictionary[rowString] = cell.ratingControl.rating
```

Comment:
I don't think this will work because of the effect we discussed of the rating control buttons not actually counting as a row select

Reply:
That will change once your task is done but as we await this and also for testing purposes i had to include it.